airbyte-integrations/connectors/source-open-exchange-rates/source_open_exchange_rates/source.py

```diff
@@ -0,0 +1,139 @@
+#
+# Copyright (c) 2022 Airbyte, Inc., all rights reserved.
+#
+
+
+from abc import ABC
+from typing import Any, Iterable, List, Mapping, MutableMapping, Optional, Tuple
+
+import pendulum
+from pendulum import DateTime
+
+import requests
+from airbyte_cdk.sources import AbstractSource
+from airbyte_cdk.sources.streams import Stream
+from airbyte_cdk.sources.streams.http import HttpStream
+from airbyte_cdk.sources.streams.http.auth import TokenAuthenticator
+
+
+class OpenExchangeRates(HttpStream, ABC):
+    url_base = "https://openexchangerates.org/api/"
+
+    primary_key = None
+    cursor_field = "date"
+
+
+    def __init__(self, base: Optional[str], start_date: str, app_id: str, **kwargs: dict) -> None:
+        super().__init__(**kwargs)
+
+        self.base = base
+        self.start_date = start_date
```

Comment:
Let's parse the date into a datetime here: 

```suggestion
        self.start_date = pendulum.parse(start_date)
```